platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SettingConstants.java

```diff
@@ -203,10 +200,22 @@ public final class SettingConstants {
             "metrics.csvOutputFolder",
             "metrics.csvFileName",
             "metrics.csvAppend",
-            "metrics.prometheusEndpointEnabled",
-            "metrics.prometheusEndpointPortNumber",
-            "metrics.prometheusEndpointMaxBacklogAllowed",
-            "metrics.disableMetricsOutput");
+            "metrics.disableMetricsOutput",
+            "prometheus.endpointEnabled",
+            "prometheus.endpointPortNumber",
+            "prometheus.endpointMaxBacklogAllowed",
+            "configPath",
+            "settingsPath",
+            "settingsUsedDir",
+            "keysDirPath",
+            "appsDirPath",
+            "logPath",
+            "paths.configPath",
```

Comment:
this is added in the new name pattern to `REMOVED_SETTINGS`?

Reply:
If someone use the new name the false warning would appear. Same for the others